components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/internal/OAuthServiceComponent.java

```diff
@@ -231,6 +232,30 @@ protected void unsetOAuthEventInterceptor(OAuthEventInterceptor oAuthEventInterc
         OAuthComponentServiceHolder.getInstance().addOauthEventInterceptorProxy(null);
     }
 
+    @Reference(
+            name = "org.wso2.carbon.identity.oauth2.token.handlers.response.AccessTokenResponseHandler",
+            service = AccessTokenResponseHandler.class,
+            cardinality = ReferenceCardinality.MULTIPLE,
+            policy = ReferencePolicy.DYNAMIC,
+            unbind = "unsetAccessTokenResponseHandler"
+    )
+    protected void setAccessTokenResponseHandler(AccessTokenResponseHandler accessTokenResponseHandler) {
+
+        if (log.isDebugEnabled()) {
+            log.info("Access token response handler impl added: " + accessTokenResponseHandler.getClass().getName());
+        }
+        OAuthComponentServiceHolder.getInstance().addAccessTokenResponseHandler(accessTokenResponseHandler);
+    }
+
+    protected void unsetAccessTokenResponseHandler(AccessTokenResponseHandler accessTokenResponseHandler) {
+
+        if (log.isDebugEnabled()) {
+            log.info("Access access token response handler removed: " +
```

Comment:
If this is an info log, we don't need to check for the `isDebugEnabled` check. Or change the info log into a debug log

Reply:
Fixed with fcc459bddf38ae1fb01650ec7c57acea167ad23c